### PR TITLE
Add two template libraries for release testing

### PR DESCRIPTION
**Pull request by JimSuplizio**
# Description
Adding two template libraries, template-two and template-three, for release testing. Template-two has template as a dependency and template-three has template and template-two as dependencies.

### CODE REVIEW

**Review by alzimmermsft — APPROVED** [review]

**Comment by alzimmermsft** [comment]
Looks like this magic: https://github.com/Azure/azure-sdk-for-java/blob/main/eng/pipelines/templates/jobs/ci.yml#L120

And other locations using `set-test-pipeline-version.yml` need to be updated with the addition libraries.

File: sdk/template/azure-sdk-template-three/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## 1.0.0-beta.1
```
Comment (alzimmermsft) [inline sdk/template/azure-sdk-template-three/CHANGELOG.md:3]:
```suggestion
## 1.0.0-beta.1 (Unreleased)
```

File: sdk/template/azure-sdk-template-two/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## 1.0.0-beta.1
```
Comment (alzimmermsft) [inline sdk/template/azure-sdk-template-two/CHANGELOG.md:3]:
```suggestion
## 1.0.0-beta.1 (Unreleased)
```

Comment (JimSuplizio) [reply sdk/template/azure-sdk-template-two/CHANGELOG.md]: Good catch @alzimmermsft. I've added a commit that fixes these.

**Pull request merged**